backend/api/utils.py

```diff
@@ -0,0 +1,10 @@
+import json
+from bson import json_util
+from typing import Any
+
+from rest_framework.response import Response
+
+
+class ApiResponse(Response):
```

Comment:
Alternative:

```python
from django.utils import simplejson

class JsonResponse(HttpResponse):
    """
        JSON response
    """
    def __init__(self, content, mimetype='application/json', status=None, content_type=None):
        super(JsonResponse, self).__init__(
            content=simplejson.dumps(content),
            mimetype=mimetype,
            status=status,
            content_type=content_type,
        )
```

Reply:
Can you also maybe rename `ApiResponse` to `JsonResponse` ?

Reply:
You can even do better (I prefer this option) : see https://docs.djangoproject.com/en/4.1/ref/request-response/#jsonresponse-objects